docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -36,38 +39,12 @@ ocp_local_config_path=/tmp/ocp
 connection=local
 ssh_cmd=""
 ----
++
+<1> Instructs os-diff to access your {OpenStackPreviousInstaller} host through SSH. The default value is `ssh -F ssh.config`. However, you can set the value without an ssh.config file, for example, `ssh -i /home/user/.ssh/id_rsa stack@my.undercloud.local`.
+<2> The host to use to access your cloud, and the podman/docker binary is installed and allowed to interact with the running containers. You can leave this key blank.
 
-Os-diff uses `ssh_cmd` to access your {OpenStackPreviousInstaller} host via SSH,
-or the host where your cloud is accessible and the podman/docker binary is installed
-and allowed to interact with the running containers. This option could have a different form:
-
-----
-ssh_cmd=ssh -F ssh.config standalone
-director_host=
-----
-
-----
-ssh_cmd=ssh -F ssh.config
-director_host=standalone
-----
-
-or without an SSH config file:
-
-----
-ssh_cmd=ssh -i /home/user/.ssh/id_rsa stack@my.undercloud.local
-director_host=
-----
-
-or
-----
-ssh_cmd=ssh -i /home/user/.ssh/id_rsa stack@
-director_host=my.undercloud.local
-----
-
-Note that the result of using `ssh_cmd` and `director_host` should be a "successful ssh access".
-
-Configure or generate the `ssh.config` file from inventory or hosts file, for example:
-
+. If you used a host file to connect to your cloud, configure the `ssh.config` file to allow os-diff to access your {OpenStackShort} environment, for example:
```

Comment:
```suggestion
. If you use a host file to connect to your cloud, configure the `ssh.config` file to allow os-diff to access your {OpenStackShort} environment, for example:
```
To keep the whole sentence in present tense, if the change doesn't make the sentence less technically accurate